fatal-errors.json

```diff
@@ -643,6 +644,12 @@
       "categoryId": "other",
       "kbLinkId": "AllFilesHaveFailed",
       "canUserFix": false
+    },
+    "DwgMissingGeometry": {
+      "description": "Entity %s is not displayed, entity handle=0x%llx.  This may or may not indicate a problem depending on whether the entity is displayed in the source CAD application.",

```

Comment:
Please make %s in double quotes to make it clear exactly what expected name should be.  Some names are plural and and be confusing.

Reply:
When I add the quotes with escape seq (\\") in the description string, the generated iModelConnectorErrors.h has the backslash removed in the entry for DwgMissingGeometry.

Somehow the python script is removing the escape character which then creates an invalid string. 
Should I add square brackets or comma instead?  
Or should I debug the python script to see if the issue can be solved.

Reply:
I'd vote single quotes to be easy.

Reply:
Please try adding two single quates instead.

Reply:
Added two single quotes.